src/Symfony/Component/Workflow/MarkingStore/MarkingStoreInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\MarkingStore;
+
+use Symfony\Component\Workflow\Marking;
+
+/**
+ * MarkingStoreInterface.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+interface MarkingStoreInterface
+{
+    public function getMarking($subject);
```

Comment:
@lyrixx you're right, it's pretty straightforward. I suppose some people will complain because their IDE won't guess the type of the return value though.